_posts/2017-11-16-cockpit-156.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Cockpit 156
+author: pitti
+date: 2017-11-16 08:00
+tags: cockpit linux
+slug: cockpit-156
+category: release
+summary: Cockpit with navigation redesign
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](http://cockpit-project.org/). We release regularly.
+Here are the release notes from version 156.
+
+### Redesign navigation and support mobile browsing
+
+The top-level dashboard menus and second-level menus of them have been
+redesigned to clarify the structure, be more extensible, and use less vertical
+screen space:
+
+![desktop mode](/images/mobilenav-desktop.png)
+
+This version introduces proper support for using on mobile phones and tablets.
+As part of this, the login screen was rearranged in mobile mode to avoid
+overlapping the on-screen keyboard with the user/password input fields:
+
+| ![mobile mode with menu](/images/mobilenav-mobile-menu.png) | &nbsp; | ![mobile login page](/images/mobilenav-login.png) |
+
+### Add project homepage link to Apps page
+
+Applications on the Apps page now show a link to the project home page, if
+provided in the AppStream metadata:
+
+![apps homepage link](/images/apps-homepage-link.png)
+
+Thanks Benjamin Deering for this feature!
+
+### Support alternate Kerberos keytabs
+
+If present, Cockpit will now use `/etc/cockpit/krb5.keytab` instead of the
```

Comment:
... Cockpit will now use the Kerberos `\etc/cockpit/krb5.keytab` file instead ...

Reply:
Updated.